common/source/docs/common-dshot-escs.rst

```diff
@@ -0,0 +1,120 @@
+.. _common-dshot-escs:
+
+==========
+DShot ESCs
+==========
+
+.. image:: ../../../images/hobbywing-esc.jpg
+
+DShot is a digital ESC protocol which allows fast, high resolution digital communication which may improve vehicle control which is especially useful in multirotor and quadplane applications.  Other advantages include
+
+- Values sent to the ESC are checksum-protected
+- Clock differences between the ESC and autopilot don't affect flight performance
+- ESC calibration is not required
+
+DShot is the underlying protocol used by :ref:`BLHeli <common-blheli32-passthru>` which offers even more features including :ref:`ESC telemetry <blheli32-esc-telemetry>`, :ref:`passthrough <common-blheli32-passthru>` and :ref:`Bi-directional dshot <bidir-dshot>`.  If choosing a DShot enabled ESC we recommend using one that also supports BLHeli or BLHeli_S.
```

Comment:
BLHeli is not a protocol. All DShot variants support telemetry and most support RPM feedback.
Passthrough is an entirely different thing to DShot. The protocol itself is actually MSP.

Reply:
Butt not all DShot ESCs have telemetry outputs...

Reply:
why are pushing that?...many dont care about ESC telemetry or even using passthrough.....but to make that accurate and not pushy:

DShot is the underlying ESC control protocol used by :ref:`BLHeli <common-blheli32-passthru>` . Many BLHeli ESC versions offer even more features such as ESC configuration, :ref:`ESC telemetry <blheli32-esc-telemetry>`, and/or  :ref:`Bi-directional dshot <bidir-dshot>`.  If choosing a DShot enabled ESC we recommend using one that also supports BLHeli32 or BLHeli_S. This allows configuration via ArduPilot's :ref:`passthrough <common-blheli32-passthru>` feature.

Reply:
This is fine with me but I think LED and Bi-directional DShot also works on non-BLHeli ESCs doesn't it?

Reply:
not actually sure that plain Dshot only ESCs have the LED control or even the bootloader to allow the bi-dir after market firmware updates....think that may be BLHeli compatible only....maybe @andyp1per knows for sure...I have had several DShot , non BLHeli ESCs in the past...non had LEDs

Reply:
changed to the suggested wording so i'll mark this as resolved.